packages/fetch-api/src/api.ts

```diff
@@ -160,7 +160,7 @@ export class GiphyFetch {
      * @returns {Promise<GifResult>}
      **/
     random(options?: RandomOptions): Promise<GifResult> {
-        return request(`${getType(options)}/random?${this.getQS(options)}`, {
+        return request(`${getType(options)}/random?${this.getQS({rating: 'pg-13', ...options})}`, {
```

Comment:
@kylebcarlson not sure why but our formatter doesn't seem to be running in your IDE or on the precommit hook. This is causing eslint to fail the builds. On Monday we'll investigate the formatter not working

Reply:
@giannif got this sorted thankfully